### PR TITLE
feat: add validation script

### PR DESCRIPTION
**Pull request by lilyhealey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# tny-archive-schemas
+
+## Validation
+
+`validate.js` is a Node.js script that validates all of the JSON files in a given folder. (It also validates individual JSON files.) It uses the [@cfworker/json-schema](https://www.npmjs.com/package/@cfworker/json-schema) NPM package to perform the validation, based on the [article.schema.json](./article.schema.json) JSON schema.
+
```
Comment (nhrqz) [inline README.md:6]: Maybe we should add some more instructions here?

```suggestion

### Requirements

- Node 22 or [nvm](https://github.com/nvm-sh/nvm)
- NPM

### Usage

If using nvm:
```bash
nvm use
\```

```bash
npm install
\```

```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+```bash
+npm run validate -- /PATH/TO/FOLDER
```
Comment (nhrqz) [inline README.md]: Seems like this also works as just `npm run validate /PATH/TO/FOLDER` ?

**Review by nhrqz — APPROVED** [review]
Looks good! Just a couple comments on the readme.

File: README.md (hunk 2 above)
Comment (lilyhealey) [reply README.md]: ooh yeah. perhaps the double dash is only necessary when using named (as opposed to positional) args.

**Pull request merged**